README.md

```diff
@@ -1 +1,140 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To Do list] <a name="list structure"></a>
+
+>**ToDo List!** is my fourth project at Microverse, in this project I have created a basic list of tasks where the user can add a new task, mark as completed and delete all the completed tasks 
+
+
+## 🛠 Built With <a name="built-with">
+
+- HTML
+- CSS
+- JavaScript
+
+</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Good HTML ,CSS and JS practices]**
+- **[This repository includes linters]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://khaled-alaa-1.github.io/To-Do-list/dist/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- browser like google chrome
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- clone the project using by running the following command :
+
+`git clone`
+`https://github.com/Khaled-Alaa-1/To-Do-list.git`
+
```

Comment:
- [ ] Please kindly use the appropriate link, what you have here is the link to the todolist, rather it should be the link to the leaderboard :+1:

Reply:
Done